### PR TITLE
Update readme for 3.7.1

### PR DESCRIPTION
**Pull request by isaacbrodsky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [![Coverage Status](https://coveralls.io/repos/github/uber/h3-java/badge.svg?branch=master)](https://coveralls.io/github/uber/h3-java?branch=master)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.uber/h3/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.uber/h3)
-[![H3 Version](https://img.shields.io/badge/h3-v3.7.0-blue.svg)](https://github.com/uber/h3/releases/tag/v3.7.0)
+[![H3 Version](https://img.shields.io/badge/h3-v3.7.2-blue.svg)](https://github.com/uber/h3/releases/tag/v3.7.2)
```
Comment (dfellis) [inline README.md:9]: Typo here?

Comment (isaacbrodsky) [reply README.md:9]: H3-Java 3.7.1 binds H3 (Core) 3.7.2.

Comment (dfellis) [reply README.md:9]: Got it.

Comment (dfellis) [reply README.md:9]: Got it.

**Review by dfellis — APPROVED** [review]

**Pull request merged**